### ISSUE TITLE
represent unmarked write-ins in the original CVR snapshot

### ISSUE THREAD

**Issue by adghayes** [votingworks/vxsuite#5779]
The original CVR snapshot has mark thresholds for all marks. When unmarked write-ins are being considered, we should similarly track write-in thresholds for all write-in areas.

Currently, unmarked write-ins are represented in a special way in the modified snapshot. This isn't quite true to the CVR spec, in which the modified snapshot should only contain marks as interpreted for tabulation. At the point of export from VxScan or VxCentralScan, those unmarked write-ins are not being considered in tabulation.

On the original snapshot, the `MarkMetricValue` field is an array and the write-in threshold could be tucked in there.

On the VxAdmin import side, the import logic will have to be updated to consider the original snapshot when looking for unmarked write-ins.

**Comment by adghayes** [comment]
I'm very open to pushback if you think this is unnecessary, but it seems to me like the appropriate way to represent unmarked write-ins as we lean into actually using the original snapshots.

**Comment by nikhilb4a** [comment]
After doing a first pass of reviewing the code, I'm wondering if this should be pushed back till after marginal marks are done. The reason being that it doesn't seem to directly impact marginal marks, unless we are choosing write-ins to surface based on write-in thresholds, though I see the connection to us using original snapshots and looking at thresholds in admin. 

We currently build out the unmarked write-ins during interpretation, and then use that when building the modified snapshot, so this change is somewhat involved. It's not especially complicated, but I'm considering sequencing it after finishing the marginal marks work. @adghayes does that make sense to you or could you fill in any context I might have missed?

**Comment by adghayes** [comment]
No, I think you're right, this is related but not a prerequisite. I had forgotten that the unmarked write-in flagging goes all the way back to the interpreter.

I think it's wise to move it down on the board to medium priority.

**Comment by adghayes** [comment]
I'm closing this because, as I write the cumulative CVR export spec, I believe that the way we represent unmarked write-ins is at a high-level correct. There are a few tweaks, but doesn't need to be tracked holistically.
